File: BackEnd/FoodOrder.Core/ViewModels/Orders/OrderVM.cs
using FoodOrder.Core.ViewModels.OrderDetails;
using FoodOrder.Core.ViewModels.OrderStatuses;
using FoodOrder.Core.ViewModels.Users;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FoodOrder.Core.ViewModels.Orders
{
    public class OrderVM
    {
        public int ID { get; set; }
        public Guid AppUserID { get; set; }
        public DateTime CreatedDate { get; set; }
        public bool IsPaid { get; set; }
        public DateTime? DatePaid { get; set; }
        public int OrderStatusID { get; set; }
        public int? PromotionID { get; set; }
        public double? PromotionAmount { get; set; }

        public string AddressString { get; set; }
        public string AddressName { get; set; }

        public UserVM UserVM { get; set; }

        public OrderStatusVM OrderStatusVM { get; set; }
        public List<OrderDetailVM> OrderDetailVMs { get; set; }
    }
}

File: bot/bot/actions/OrderVM.py
class OrderVM:

    def __init__(self, id, appUserID, createdDate, isPaid, datePaid, orderStatusID, promotionID, finalTotalPrice, promotionAmount, addressString,
        addressName, userVM, orderStatusVM, orderDetailVMs
    ):
        self.id = id
        self.appUserID = appUserID
        self.createdDate = createdDate
        self.isPaid = isPaid
        self.datePaid = datePaid
        self.orderStatusID = orderStatusID
        self.promotionID = promotionID
        self.finalTotalPrice = finalTotalPrice
        self.promotionAmount = promotionAmount
        self.addressString = addressString,
        self.addressName = addressName
        self.userVM = userVM
        self.orderStatusVM = orderStatusVM
        self.orderDetailVMs = orderDetailVMs
        